build/android/native/build.gradle

```diff
@@ -0,0 +1,60 @@
+apply plugin: 'com.android.library'
+import org.ajoberstar.grgit.Grgit
+
+android {
+	compileSdkVersion 29
+	buildToolsVersion '29.0.2'
```

Comment:
This line is obsolete

Reply:
Both projects must contain the same version. Otherwise, gradle will start downloading the latest version. We do not necessarily want this.